appservice/src/main/resources/static/swagger/swagger-spec.yaml

```diff
@@ -438,6 +438,41 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/users/{userID}/activity/days/':
+    parameters:
+      - name: userID
+        in: path
+        description: The ID of the user
+        required: true
+        type: string
+      - name: Yona-Password
+        in: header
+        description: The device password
+        required: true
+        type: string
+    get:
+      summary: Get day network and app activity
+      description: Get activity of the current day, pageable to yesterday and before 
+      parameters:
+        - name: size
+          in: query
+          description: The number of elements to be fetched
+          required: false
+          default: 10
```

Comment:
Make this 3, in accordance with the [functional requirements](http://wiki.yona.nu/display/DEV/Me+-+Day)